sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -498,9 +506,6 @@ object Radar {
             this.logger.d("Using Huawei location services")
         }
 
-        val application = this.context as? Application
```

Comment:
Looks like this one may have gotten lost in the shuffle (and we can probably remove the new member var up top)

Reply:
good catch, just fixed and pushed another orange